libs/knowledge-store/ragstack_knowledge_store/graph_store.py

```diff
@@ -559,6 +582,8 @@ def similarity_search(
     def _get_adjacent(
```

Comment:
Annoyingly, `_get_adjacent` is also used in the testing method `_linked_ids`. I propose we get rid of it. Then I wouldn't need the version without embedings, since it is only used by MMR.